homeassistant/components/ld2410_ble/coordinator.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, hass: HomeAssistant, ld2410_ble: LD2410BLE) -> None:
     def _async_handle_update(self, state: LD2410BLEState) -> None:
         """Just trigger the callbacks."""
         self.connected = True
-        self.async_set_updated_data(True)
+        self.async_set_updated_data(None)
```

Comment:
`coordinator.data` isn't used, so I set it to `None`. Alternatively I could change the subscript for `DataUpdateCoordinator` to `bool`.